sql/src/test/java/io/crate/planner/operators/PushDownTest.java

```diff
@@ -50,24 +54,17 @@ public void testOrderByOnUnionIsMovedBeneathUnion() {
         assertThat(plan, isPlan(sqlExecutor.functions(),  "Boundary[name]\n" +
                                                 "Union[\n" +
                                                     "Boundary[name]\n" +
-                                                    "OrderBy[name]\n" +
+                                                    "OrderBy[name][false][null]\n" +
                                                     "Collect[doc.users | [name] | All]\n" +
                                                 "---\n" +
                                                     "Boundary[text]\n" +
-                                                    "OrderBy[text]\n" +
+                                                    "OrderBy[text][false][null]\n" +
                                                     "Collect[doc.users | [text] | All]\n" +
                                                 "]\n"));
     }
 
     @Test
     public void testOrderByOnUnionIsCombinedWithOrderByBeneathUnion() {
```

Comment:
This name doesn't apply anymore.

Reply:
The test should still work as the union doesn't use an actual fetch operation.

Reply:
Hmm, but we don't know that the Union is not gonna use the fetch, or we do somehow?